server/events/models/models.go

```diff
@@ -108,6 +108,9 @@ type PullRequest struct {
 	// Gitlab supports an additional "merged" state but Github doesn't so we map
 	// merged to Closed.
 	State PullRequestState
+
+	// The repository the pull request is for
```

Comment:
- We need to include the `BaseRepo` and `HeadRepo` since if it's a fork PR these are different.
- Comments for fields/methods/structs need to start with the name for them to work in godoc. They should also end with a period as per golang conventions.
```
BaseRepo is the repository that this pull request is being pulled into.
```